aiohttp/helpers.py

```diff
@@ -82,6 +82,10 @@ def noop(*args, **kwargs):  # type: ignore
     return  # type: ignore
 
 
+async def noop2(*args: Any, **kwargs: Any) -> None:
```

Comment:
Let's rename it to represent the intent:
```suggestion
async def async_noop(*args: Any, **kwargs: Any) -> None:
```

Reply:
1. Both noops are async.
2. I don't care too much, the function is internal and will be removed in aiohttp 4.